x-pack/platform/plugins/shared/actions/server/usage/actions_telemetry.ts

```diff
@@ -109,12 +109,29 @@ export async function getTotalCount(
     };
   } catch (err) {
     const errorMessage = err && err.message ? err.message : err.toString();
+    let returnedErrorMessage = errorMessage;
+    const errorStr = JSON.stringify(err);
+    const logMessage = `Error executing actions telemetry task: getTotalCount - ${err}`;
```

Comment:
Should we use `${errorStr}` rather than `${err}` here?

Reply:
I thought using `err` would return the whole error with stack trace, therefore it would pollute the data as we already store the stack trace separately.

If this is not the case, ok :) 

We can try `errorMessage` as well but probably would return the same thing.